### PR TITLE
Fix OADP-4623: OpenShift on IBMCloud setup for OADP (#1482)

### PR DESCRIPTION
**Pull request by shubham-pampattiwar**
(cherry picked from commit b8fb9ddedca10bc0d77e65f9f6c776ef941a972f)

### CODE REVIEW

**Review by kaovilai — APPROVED** [review]

File: hack/extra-crds/config-operator_01_infrastructures-Default.crd.yaml
Comment (kaovilai) [inline hack/extra-crds/config-operator_01_infrastructures-Default.crd.yaml:1]: This is just for unit tests correct?

Comment (shubham-pampattiwar) [reply hack/extra-crds/config-operator_01_infrastructures-Default.crd.yaml:1]: bingo

**Review by sseago — APPROVED** [review]

**Pull request merged**